packages/ui-extensions/src/surfaces/admin/components/AdminAction.d.ts

```diff
@@ -48,66 +52,53 @@ export interface ClickOptions {
-  /** @private */
-  attributeChangedCallback(name: string): void;
-  /** @private */
-  connectedCallback(): void;
-  /** @private */
-  disconnectedCallback(): void;
-  /** @private */
-  adoptedCallback(): void;
-  /**
-   * Queue a run of the render function.
-   * You shouldn't need to call this manually - it should be handled by changes to @property values.
-   * @private
-   */
-  queueRender(): void;
-  /**
-   * Like the standard `element.click()`, but you can influence the behavior with a `sourceEvent`.
-   *
-   * For example, if the `sourceEvent` was a middle click, or has particular keys held down,
-   * components will attempt to produce the desired behavior on links, such as opening the page in the background tab.
-   * @private
-   * @param options
-   */
-  click({sourceEvent}?: ClickOptions): void;
+        /** @private */
+    static get observedAttributes(): string[];
+    constructor({ styles, ShadowRoot: renderFunction, delegatesFocus, ...options }: RenderImpl);
+    /** @private */
+    setAttribute(name: string, value: string): void;
+    /** @private */
+    attributeChangedCallback(name: string): void;
+    /** @private */
+    connectedCallback(): void;
+    /** @private */
+    disconnectedCallback(): void;
+    /** @private */
+    adoptedCallback(): void;
+    /**
+     * Queue a run of the render function.
+     * You shouldn't need to call this manually - it should be handled by changes to @property values.
+     * @private
+     */
+    queueRender(): void;
+    /**
+     * Like the standard `element.click()`, but you can influence the behavior with a `sourceEvent`.
+     *
+     * For example, if the `sourceEvent` was a middle click, or has particular keys held down,
+     * components will attempt to produce the desired behavior on links, such as opening the page in the background tab.
+     * @private
+     * @param options
+     */
+    click({ sourceEvent }?: ClickOptions): void;
 }
 
-declare class AdminAction
-  extends PreactCustomElement
-  implements AdminActionProps
-{
-  heading: string;
-  loading: boolean;
-  constructor();
+declare class AdminAction extends PreactCustomElement implements AdminActionProps {
+    heading: string;
+    loading: boolean;
+    constructor();
 }
 declare global {
-  interface HTMLElementTagNameMap {
-    [tagName]: AdminAction;
-  }
+    interface HTMLElementTagNameMap {
+        [tagName]: AdminAction;
+    }
 }
 declare module 'preact' {
-  namespace createElement.JSX {
-    interface IntrinsicElements {
-      [tagName]: Omit<
-        HTMLAttributes<HTMLElement>,
-        Extract<keyof HTMLAttributes<HTMLElement>, `on${Capitalize<string>}`>
-      > &
-        Omit<AdminActionJSXProps, 'primaryAction' | 'secondaryActions'>;
+    namespace createElement.JSX {
+        interface IntrinsicElements {
+            [tagName]: Omit<HTMLAttributes<HTMLElement>, Extract<keyof HTMLAttributes<HTMLElement>, `on${Capitalize<string>}`>> & AdminActionJSXProps;
+        }
```

Comment:
This should be `Omit<AdminActionJSXProps, 'primaryAction' | 'secondaryActions'>`

Curious why this is missing now